programs/move_loader/src/lib.rs

```diff
@@ -4,10 +4,9 @@ pub mod error_mappers;
 pub mod processor;
 
 use crate::processor::process_instruction;
-use solana_sdk::move_loader;
 
 solana_sdk::declare_program!(
-    move_loader::BS58_STRING,
+    "MoveLdr111111111111111111111111111111111111",
```

Comment:
Can we avoid duplicating the definition of the id string between here and the sdk?

Reply:
Yup, done